src/vm/wren_core.c

```diff
@@ -666,6 +667,21 @@ DEF_PRIMITIVE(num_sign)
   }
 }
 
+DEF_PRIMITIVE(num_highest)
+{
+  RETURN_NUM(DBL_MAX);
+}
+
+DEF_PRIMITIVE(num_lowest)
+{
+  RETURN_NUM(-DBL_MAX);
```

Comment:
My idea was for `highest` to correspond to the "largest" positive number, and `lowest` to correspond to the "largest" negative number. Note that `DBL_MIN` corresponds to the smallest positive normalized number (the negative sign is in the exponent).